tools/normal_estimation.cpp

```diff
@@ -135,21 +135,21 @@ batchProcess (const std::vector<std::string> &pcd_files, std::string &output_dir
 #pragma omp parallel for \
   default(none) \
   shared(k, output_dir, pcd_files, radius)
-  for (int i = 0; i < static_cast<int>(pcd_files.size ()); ++i)
+  for (const auto & pcd_file : pcd_files)
```

Comment:
OpenMP supports range-for loops only in very recent versions, so let's keep the "traditional" for-loop to stay compatible with older OpenMP versions (one more in fast_bilateral_filter.cpp)